File: app/src/main/java/com/unionpay/numberpassword/MainActivity.java
package com.unionpay.numberpassword;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Button;

import com.unionpay.numberpassword.NumberPassword.NumberPassword;
import com.unionpay.numberpassword.RememberNumbers.RememberNumbers_main;

import java.util.List;

/**
 * Created by hzqb_luke on 30/04/2017.
 */

public class MainActivity extends AppCompatActivity {
    Button button1,button2;

    private RecyclerView mRecyclerView;
    private List<Integer> mDatas;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
//        this.requestWindowFeature(Window.FEATURE_NO_TITLE);


        button1 = (Button) findViewById(R.id.button_nbpass);
        button1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,NumberPassword.class));
            }
        });

        button2 = (Button) findViewById(R.id.button_remnumber);
        button2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,RememberNumbers_main.class));
            }
        });
    }
}

File: app/src/main/java/com/unionpay/numberpassword/MainActivity.kt
package com.unionpay.numberpassword

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import com.unionpay.numberpassword.NumberPassword.NumberPassword
import com.unionpay.numberpassword.RememberNumbers.RememberNumbers_main
import com.unionpay.numberpassword.RememberPoker.RememberPoker

/**
 * Created by hzqb_luke on 30/04/2017.
 */

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)

        val button1 = findViewById(R.id.button_nbpass) as Button
        button1.setOnClickListener { startActivity(Intent(this@MainActivity, NumberPassword::class.java)) }

        var button2 = findViewById(R.id.button_remnumber) as Button
        button2.setOnClickListener { startActivity(Intent(this@MainActivity, RememberNumbers_main::class.java)) }

        var button3 = findViewById(R.id.button_rmPoker) as Button
        button3.setOnClickListener { startActivity(Intent(this@MainActivity, RememberPoker::class.java)) }
    }
}
